src/Value/Color.php

```diff
@@ -233,6 +233,10 @@ public function render(OutputFormat $outputFormat): string
             return $this->renderAsHex();
```

Comment:
Should we move these three conditions into a method `shouldRenderInLegacySyntax`?

Reply:
That's a good idea.  Though it's "hex" rather than "legacy" syntax in this case ("legacy" is with commas between the arguments; "modern" is with spaces between the argumens but a slash separating the alpha argument; "hex" is the notation that begins `#` - it's a complete mess IMO, but those are the standards).

But it's beyond the scope of this PR, so I've added #852.